### PR TITLE
Admin metrics dashboard page

### PR DESCRIPTION
**Pull request by Ennyola**

### CODE REVIEW

File: src/routes/routes.js [hunk 1]
```diff
@@ -1,131 +1,133 @@
...
+    app.get('/post', (req, res) => res.render('userPost', { isAuth: true }));
+    app.get('/jobInvites', renderJobInvitesPage);
+
+    app.get('/post/:inviteId', renderSinglePostPage);
+
+    // Edit post endpoint
+    app.get('/post/:inviteId/edit', validateInviteId, validateInvite, editInvite);
+
+    app.get('/admin/users', renderAdminUsersPage);
+    app.get('/admin/metricsDashboard', (req, res) => res.render('metricsDashboard', { isAuth: false }));
```
Comment (shonubijerry) [inline src/routes/routes.js]: let this be
`app.get('/admin', (req, res) => res.render('metricsDashboard', { isAuth: false }));`

This means that users should be taken to the admin page with that route.

File: src/views/metricsDashboard.ejs [hunk 2]
```diff
@@ -0,0 +1,145 @@
+<% include partials/header %>
```
Comment (shonubijerry) [inline src/views/metricsDashboard.ejs]: Let this file name be `./src/views/admin/index.ejs`

**Review by shonubijerry — CHANGES_REQUESTED** [review]

File: src/routes/routes.js (hunk 1 above)
Comment (Ennyola) [reply src/routes/routes.js]: it has been resolved

File: src/views/metricsDashboard.ejs (hunk 2 above)
Comment (Ennyola) [reply src/views/metricsDashboard.ejs]: I've done this too
Want to begin working on the responsiveness of the page

**Pull request merged**